### ISSUE TITLE
convert problems with paths containing spaces

### ISSUE THREAD

**Issue by blhiggins** [hallowelt/migrate-confluence#71]
Hi,

I'm attempting my first conversion, with my data located under the path "/home/b2higgin/wiki snippets/wiki parse/". When I run the migrate-confluence convert command I find several instances of "pandoc: /home/b2higgin/wiki: openBinaryFile: does not exist (No such file or directory)" are shown in the console output. Does not trigger for every file. If I change spaces to underscores in the path and re-run convert, those messages disappear. Sounds like the path is not being fully quoted at some point.

Best,
Brendon

**Comment by osnard** [comment]
You are right. The issue is here. https://github.com/hallowelt/mediawiki-lib-migration/blob/master/src/Converter/PandocHTML.php#L15

**Comment by Boldewyn** [comment]
Fixed in https://github.com/hallowelt/mediawiki-lib-migration/pull/25, needs a merge + lib-update.
